src/mono/wasm/test-main.js

```diff
@@ -45,6 +51,12 @@ if (!is_node && !is_browser && typeof globalThis.crypto === 'undefined') {
     }
 }
 
+if (ENVIRONMENT_IS_WEB && !ENVIRONMENT_IS_WORKER) {
+    console.log(" ---------------- ");
+    console.log("Running at: " + globalThis.location.href);
```

Comment:
is this meant to be checked in?

Reply:
yes, but maybe we don't need the ASCII art :)